server/start.go

```diff
@@ -247,7 +247,9 @@ func startInProcess(ctx *Context, clientCtx client.Context, appCreator types.App
 
 	config := config.GetConfig(ctx.Viper)
 	if err := config.ValidateBasic(); err != nil {
-		return err
+		ctx.Logger.Error("WARNING: The minimum-gas-prices config in app.toml is set to the empty string. " +
```

Comment:
oh, we don't have `Warn` method.

Reply:
We could add it to the interface. ERR is fine for now IMHO.